xbmc/settings/lib/SettingSection.cpp

```diff
@@ -78,6 +81,20 @@ bool CSettingGroup::Deserialize(const TiXmlNode *node, bool update /* = false */
   if (!ISetting::Deserialize(node, update))
     return false;
 
+  const TiXmlElement *element = node->ToElement();
+  if (element == NULL)
+    return false;
+
+  int tmp = -1;
+  if (element->QueryIntAttribute(SETTING_XML_ATTR_LABEL, &tmp) == TIXML_SUCCESS && tmp > 0)
+    m_label = tmp;
+  if (element->QueryIntAttribute(SETTING_XML_ATTR_HELP, &tmp) == TIXML_SUCCESS && tmp > 0)
+    m_help = tmp;
+  if (element->QueryIntAttribute(SETTING_XML_ATTR_BEFORE, &tmp) == TIXML_SUCCESS && tmp > 0)
```

Comment:
You can't abuse `SETTING_XML_ATTR_BEFORE` here since it's already being used to be able to insert a group into a specific place.